frontend/packages/ceph-storage-plugin/src/components/block-pool/block-pool-details-page.tsx

```diff
@@ -3,14 +3,20 @@ import { useTranslation } from 'react-i18next';
 
 import { DetailsPage } from '@console/internal/components/factory';
 import { navFactory } from '@console/internal/components/utils';
+import { useK8sWatchResource } from '@console/internal/components/utils/k8s-watch-hook';
 
 import { menuActionCreator } from './block-pool-menu-action';
+import { CephClusterKind } from '../../types';
+import { cephClusterResource } from '../../resources';
 import { BlockPoolDashboard } from '../dashboards/block-pool/block-pool-dashboard';
 
 const BlockPoolDetailsPage: React.FC<React.ComponentProps<typeof DetailsPage>> = (props) => {
   const { editYaml } = navFactory;
   const { t } = useTranslation();
 
+  // Hooks
```

Comment:
```suggestion
```
nit: Any function with `use` is a Hook. No need for the comment.

Reply:
removed